src/main/java/uk/gov/hmcts/reform/managecase/repository/DefaultDataStoreRepository.java

```diff
@@ -110,13 +118,32 @@ public void removeCaseUserRoles(List<CaseUserRole> caseUserRoles, String organis
         dataStoreApi.removeCaseUserRoles(new CaseUserRolesRequest(caseUsers));
     }
 
+    @Override
+    public CaseUpdateViewEvent getStartEventTrigger(String caseId, String eventId) {
+        String userAuthToken = getUserAuthToken();
+        return dataStoreApi.getStartEventTrigger(userAuthToken, caseId, eventId);
+    }
+
+    @Override
+    public StartEventResource getExternalStartEventTrigger(String caseId, String eventId) {
+        String userAuthToken = getUserAuthToken();
+        return dataStoreApi.getExternalStartEventTrigger(userAuthToken, caseId, eventId);
+    }
+
+    @Override
+    public CaseResource submitEventForCaseOnly(String caseId, CaseDataContent caseDataContent) {
```

Comment:
Can this be renamed to remove the word "Only"

`submitEventForCase` will be renamed as part of code review changes for ACA-68 in #150

Reply:
[re-raised](#discussion_r533373203) during a re-review of current code.  So resolving this earlier conversation.